Shopipy.ProductManagement/Services/ProductService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Shopipy.Persistence.Models;
+using Shopipy.Persistence.Repositories;
+using Shopipy.Shared.Services;
+
+namespace Shopipy.ProductManagement.Services
+{
+    public class ProductService : IProductService
```

Comment:
You could use primary constructors here, Konstantinas might be very angry about not using them

Reply:
done